packages/host/tests/integration/components/operator-mode-test.gts

```diff
@@ -568,6 +570,112 @@ module('Integration | operator-mode', function (hooks) {
     };
   });
 
+  test<TestContextWithSave>('it allows chat commands to change cards in the stack', async function (assert) {
```

Comment:
if you aren't using `this.onSave` there is no need to parameterize the type on this test.

Reply:
that being said--should we be using `this.onSave()` to assert that the AI bot is modifying the cards?

Reply:
Removed it for the test that doesn't change a card, added an onSave assertion for the one that should change the card